prometheus/histogram.go

```diff
@@ -183,11 +183,13 @@ func newHistogram(desc *Desc, opts HistogramOpts, labelValues ...string) Histogr
 		opts.Buckets = DefBuckets
 	}
 
+	// one memory alloctation
+	var bothCounts [2]histogramCounts
```

Comment:
Not sure this is worth it. It is not in the hot path and thus most likely will never have a measurable impact. On the other hand, it requires an additional line of code, a line of explanatory comment, and makes the `counts` initialization less readable.

Reply:
Reasonable argument. I can revert.